<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newClient</class>
 <widget class="QDialog" name="newClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>358</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>MS Sans Serif</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Registration for New Customers </string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
    background-color:	#3b3b3b;

	color: #fff;
	selection-background-color: #fff;
	selection-color: #000;

}


/*-----QLabel-----*/
QLabel
{
	background-color: transparent;
	color: #fff;

}


/*-----QPushButton-----*/
QPushButton
{
	background-color: #00aae4;
	color: #fff;
	min-width: 80px;
	border-radius: 4px;
	padding: 5px;

}



QPushButton::hover
{
	background-color: #0486b2;
	border: 1px solid #0486b2;

}



/*-----QLineEdit-----*/
QLineEdit
{
	background-color: #242424;
	color : #fff;
	border: 1px solid #1d1d1d;
	padding: 3px;
	padding-left: 5px;
	border-radius: 4px;
   font-weight: bold;
}

/*-----QRadioButton-----*/
QRadioButton 
{
	color: #fff;
	background-color: transparent;

}


QRadioButton::indicator::unchecked:hover 
{
	background-color: #d3d3d3;
	border: 2px solid #002b2b;
	border-radius: 6px;
}


QRadioButton::indicator::checked 
{
	border: 2px solid  #d3d3d3;
	border-radius: 6px;
	background-color:  #0486b2;  
	width: 9px; 
	height: 9px; 

}





</string>
  </property>
  <widget class="QLabel" name="welcome_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Registration Form</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newAccount_Save">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Save </string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>289</width>
     <height>260</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox 
{
   background-color:#838996;
    border: 1px solid;
    border-color: #1d1d1d;
	border-radius: 4px;
    margin-top: 23px;

}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="firstname">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>61</y>
      <width>129</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lastname">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>91</y>
      <width>129</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="accNum">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>31</y>
      <width>129</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="newUsername">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>175</y>
      <width>129</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="newPassword">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>205</y>
      <width>129</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QRadioButton" name="current">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>123</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Current</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="saving">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>123</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Savings</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="balance">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>145</y>
      <width>129</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>33</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
 background-color:#838996;
}</string>
    </property>
    <property name="text">
     <string>Account Number         </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>63</y>
      <width>92</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
 background-color:#838996;
}</string>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>92</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
 background-color:#838996;
}</string>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
 background-color:#838996;
}</string>
    </property>
    <property name="text">
     <string>Account Type </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>149</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
 background-color:#838996;
}</string>
    </property>
    <property name="text">
     <string>Balance </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>177</y>
      <width>86</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
 background-color:#838996;
}</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>206</y>
      <width>83</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
 background-color:#838996;
}</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_8</zorder>
   <zorder>label_7</zorder>
   <zorder>label</zorder>
   <zorder>firstname</zorder>
   <zorder>lastname</zorder>
   <zorder>accNum</zorder>
   <zorder>newUsername</zorder>
   <zorder>newPassword</zorder>
   <zorder>current</zorder>
   <zorder>saving</zorder>
   <zorder>balance</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
